modules/benchmark/R/calc.benchmark.R

```diff
@@ -0,0 +1,123 @@
+##-------------------------------------------------------------------------------------------------#
+##' For each benchmark id, calculate metrics and update benchmarks_ensemble_scores
+##'  
+##' @name calc.benchmark 
+##' @title Calculate benchmarking statistics
+##' @param settings object from start.benchmark.runs (for now - we many only need a few variables)
+##' @param con database connection
+##' @export 
+##' 
+##' @author Betsy Cowdery 
+
+calc.benchmark <- function(settings, con){ #settings file is output from start.benchmark.runs
+  
+  require(RPostgreSQL)
+  require(XML)
+  require(data.table)
```

Comment:
where is the `XML` and `data.table` packages used (if these are used in a function called within, wouldn't it make more sense to use the `require` section there?